lib/matplotlib/tests/test_axes.py

```diff
@@ -6758,3 +6759,13 @@ def test_2dcolor_plot(fig_test, fig_ref):
     axs[2].step([1, 2], [1, 2], c=color.reshape((1, -1)))
     axs[3].hist(np.arange(10), color=color.reshape((1, -1)))
     axs[4].bar(np.arange(10), np.arange(10), color=color.reshape((1, -1)))
+
+
+@pytest.mark.parametrize('ha', ['left', 'center', 'right'])
+def test_ylabel_ha_with_position(ha):
+
```

Comment:
Any reason for the blank line? (It just sticks out from all the other test functions.)

Reply:
Just being sloppy...